poanetwork/token-wizard
# (Bug) Keep data between the steps (Manage Page)

This is an integration PR for the manage section

How to test it and what to test:
- Go to the home
- Create a crowdsale (Minted or Dutch)
- Go to the home, choose the created crowdsale
- You should see the created crowdsale
- If you refresh the page, you should see the data again
